js/poloniex.js

```diff
@@ -1353,14 +1353,14 @@ module.exports = class poloniex extends Exchange {
         const asksResult = [];
         const bidsResult = [];
         for (let i = 0; i < asks.length; i++) {
-            if ((i % 2) === 0) {
+            if (i % 2) {
```

Comment:
@sc0Vu in this case you've also inverted the conditional, no?

Reply:
oh, I think you're right!